plugins/ida/ida_main.ml

```diff
@@ -135,29 +135,17 @@ let main () =
   register_source (module Reconstructor);
   Project.Input.register_loader name loader
 
-module Main = struct
-  open Cmdliner
-  let man = [
-    `S "DESCRIPTION";
-    `P "This plugin provides rooter, symbolizer and reconstuctor services.";
-    `P "If IDA instance is found on the machine, or specified by a
-  user, it will be queried for the specified information."
-
-  ]
-
-
-  let path : string option Term.t =
-    let doc = "Use IDA to extract symbols from file. \
-               You can optionally provide path to IDA executable,\
-               or executable name." in
-    Arg.(value & opt (some string) None & info ["path"] ~doc)
-
-  let info = Term.info name ~version ~doc ~man
-
-  let () =
-    let run = Term.(const main $ const ()) in
-    match Term.eval ~argv ~catch:false (run,info) with
-    | `Ok () -> ()
-    | `Help | `Version -> exit 0
-    | `Error _ -> exit 1
-end
+let () =
+  let () = Config.manpage [
+      `S "DESCRIPTION";
+      `P "This plugin provides rooter, symbolizer and reconstuctor services.";
+      `P "If IDA instance is found on the machine, or specified by a
+        user, it will be queried for the specified information."
+    ] in
+  let path = Config.(param (some string) "path" ~default:None
+                       ~doc:"Use IDA to extract symbols from file. \
+                             You can optionally provide path to IDA \
+                             executable, or executable name.") in
+  Config.when_ready (fun {Config.get=(!)} ->
+      (* TODO: Use [path] when calling IDA. *)
```

Comment:
Such code shouldn't be in the master branch. We shouldn't merge a code the definitely doesn't work. It is better to make a separate PR where path is implemented.